config/dev.webpack.config.js

```diff
@@ -44,35 +44,38 @@ plugins.push(
       exposes: {
         // Application root
         './RootApp': resolve(__dirname, '../src/AppEntry'),
+        // expose Images detail to be used on insights
+        './Images': resolve(__dirname, '../src/Routes/ImageManager/Images.js'),
+        './ImagesSetTable': resolve(
+          __dirname,
+          '../src/Routes/ImageManager/ImageSetsTable.js'
+        ),
+        './ImagesDetail': resolve(
+          __dirname,
+          '../src/Routes/ImageManagerDetail/ImageDetail.js'
+        ),
         // expose System detail to be used on insights
         './Inventory': resolve(__dirname, '../src/Routes/Devices/Inventory.js'),
         './DeviceTable': resolve(
           __dirname,
           '../src/Routes/Devices/DeviceTable.js'
         ),
-        // expose Images to use in Image Builder
-        './ImagesTable': resolve(
+        './UpdateDeviceModal': resolve(
+          __dirname,
+          '../src/Routes/Devices/UpdateDeviceModal.js'
+        ),
+        './UpdateSystem': resolve(
           __dirname,
-          '../src/Routes/ImageManager/ImageTable.js'
+          '../src/Routes/Devices/UpdateSystem.js'
         ),
-        // './UpdateDeviceModal': resolve(
-        //   __dirname,
-        //   '../src/Routes/Devices/UpdateDeviceModal.js'
-        // ),
-        // './AddDeviceModal': resolve(
-        //   __dirname,
-        //   '../src/Routes/Devices/AddDeviceModal.js'
-        // ),
-        // './CreateGroupModal': resolve(
-        //   __dirname,
-        //   '../src/Routes/Groups/CreateGroupModal.js'
-        // ),
-        // './RemoveDeviceModal': resolve(
-        //   __dirname,
-        //   '../src/Routes/Devices/RemoveDeviceModal.js'
-        // ),
       },
-      shared: [{ 'react-redux': { requiredVersion: deps['react-redux'] } }],
+      shared: [
+        { 'react-redux': { requiredVersion: deps['react-redux'] } },
+        { 'react-dom': { singleton: true, eager: true } },
+        { 'react-router-dom': { singleton: true, requiredVersion: '*' } },
```

Comment:
This line is causing edge-frontend to fail to load. The error it returns is : 
```
Uncaught Error: You cannot render a <Router> inside another <Router>. You should never have more than one in your app
```
The solution I found is replacing this line with this:
```suggestion
        { 'react-router-dom': { requiredVersion: deps['react-router-dom'] } },
```